.github/workflows/async-auto-label.yml

```diff
@@ -12,7 +15,8 @@ jobs:
     name: Apply label
     runs-on: ubuntu-latest
     if: ${{ github.event.issue.pull_request &&
-            startsWith('+async-label', github.event.comment.body) }}
+            (startsWith(github.event.comment.body, '+label-async-') ||
+             startsWith(github.event.comment.body, '!+label-async-')) }}
```

Comment:
I will check the association already before running python. That removes some overhead